### PR TITLE
feat: European Union countries list

### PR DESCRIPTION
**Pull request by leamotta**
Add European Union countries list. It will be used to determine shipping rules in Europe (domestic vs rest of world)

### CODE REVIEW

File: spec/commerce_helpers_spec.rb [hunk 1]
```diff
@@ -49,4 +49,8 @@
   it 'has a list of countries in continental Europe' do
     expect(CommerceHelpers::COUNTRIES_IN_CONTINENTAL_EUROPE).not_to be_empty
   end
+
+  it 'has a list of countries in the European Union' do
+    expect(CommerceHelpers::COUNTRIES_IN_EUROPEAN_UNION).not_to be_empty
+  end
```
Comment (starsirius) [inline spec/commerce_helpers_spec.rb:55]: #non-blocking. I can't recognize the 2-letter country code easily so I tried to be [more explicit](https://github.com/artsy/commerce_helpers/blob/92a77d3e13741cf416e799a2f2d9dd7f99b35a77/spec/commerce_helpers_spec.rb#L10-L47) in the test (and can be treated as the spec as well).

**Review by starsirius — APPROVED** [review]
Woohoo! Excited! 🎉

File: lib/commerce_helpers.rb [hunk 2]
```diff
@@ -7,4 +7,6 @@ module CommerceHelpers
                                SK].freeze
   COUNTRIES_IN_CONTINENTAL_EUROPE = %w[AD AM AT AZ BA BE BG BY CH CY CZ DE DK EE ES FI FR GE HR HU IT KZ LI LT LU LV MC
                                        MD ME MK NL NO PL PT RO RS RU SE SI SK SM TR UA VA].freeze
+  COUNTRIES_IN_EUROPEAN_UNION = %w[AT BE BG CY CZ DE DK EE ES FI FR GR HR HU IE IT LT LU LV MT NL PL PT RO SE SI SK]
```
Comment (oxaudo) [inline lib/commerce_helpers.rb:10]: Same as @starsirius - wondering if for our sanity would be cool to have a list of full name in comment here.

**Review by oxaudo — APPROVED** [review]
Looks great to me! Do think that it would be cool to have a full list somewhere as well. Tests or just comment on the actual object.

**Pull request merged**